stellar/deploy-contract.js

```diff
@@ -221,7 +149,7 @@ async function deploy(options, config, chain, contractName) {
         return;
     }
 
-    const wasmHash = await uploadContract(contractName, options, wallet, chain);
+    const wasmHash = await uploadWasm(options.contractCodePath, wallet, chain);
```

Comment:
```suggestion
    const wasmHash = await uploadContractCode(wallet, chain, options);
```
abstract it away so `options.contractCodePath` doesn't need to be referenced directly since it's not actually a cli option

Reply:
done